### PR TITLE
docs: k8s: updating docs for k8s v1.9, 1.10 and 1.11 support

### PR DESCRIPTION
**Pull request by tasdikrahman**
updates docs

### CODE REVIEW

**Comment by tgraf** [comment]
test-docs-please

**Review by tgraf — APPROVED** [review]

File: Documentation/kubernetes/compatibility.rst [hunk 1]
```diff
@@ -15,7 +15,9 @@ All Kubernetes versions listed are compatible with Cilium:
 ============= =========================== ============================
  1.7           * `extensions/v1beta1`_    ``cilium.io/v2`` has a
  1.8           * `networking.k8s.io/v1`_  `CustomResourceDefinition`
- 1.9
```
Comment (aanm) [inline Documentation/kubernetes/compatibility.rst]: The problem with the table is that each line written in the rst will be a real line after rendering.

I would suggested to put it like:

```
+----------------------+---------------------------+----------------------------+
| k8s Version          | k8s NetworkPolicy API     | CiliumNetworkPolicy        |
+----------------------+---------------------------+----------------------------+
| 1.7                  | * `extensions/v1beta1`_   | ``cilium.io/v2`` has a     |
+----------------------+---------------------------+ `CustomResourceDefinition` |
| 1.8, 1.9, 1.10, 1.11 | * `networking.k8s.io/v1`_ |                            |
+----------------------+---------------------------+----------------------------+
```

Comment (tasdikrahman) [reply Documentation/kubernetes/compatibility.rst]: @aanm addressed your concern :)

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
test-docs-please

**Comment by borkmann** [comment]
Thanks a lot @tasdikrahman!

**Pull request merged**